### DISCUSSION TITLE
Simulator arquitecture is unsupported mac

### DISCUSSION THREAD

**Discussion by AlexBravo95** [appium/appium#17991] [General]
Hi,
I am having the following error, I am wondering if it is the mac or the xcode, I was validating the .ipa on several macs and it shows the same message.

I did the test on browserstack and it works fine.

According to my theory it has to do with the x64 or x86 architecture, or that development compiles it in some other way

¿any ideas?, thanks

![image](https://user-images.githubusercontent.com/77693511/209989941-533737e9-9a67-434f-a697-ab6150856376.png)

![image](https://user-images.githubusercontent.com/77693511/209989522-a503fa69-4393-40ab-bff6-178ff2df15d6.png)

**Comment by KazuCocoa** [comment]
You may need to prepare a build for a simulator, or for x86 architecture

**Reply by AlexBravo95** [reply]
I have the same idea in mind, it goes on the compilation side
